site/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "hacker-news-pwas",
+  "description": "Hacker News readers as Progressive Web Apps",
+  "scripts": {
+    "build": "bundle exec jekyll build",
+    "dev": "bundle exec jekyll serve",
+    "dev-sw": "npm run build && sw-precache --config=sw-precache-config.js && npm run dev",
```

Comment:
Oooooooooh yes you're right. Thanks for testing it thoroughly that makes perfect sense actually.